docs/maintainer/infrastructure.md

```diff
@@ -2,28 +2,155 @@
+
+#### conda-forge-repodata-patches
+
+This repository creates the `repodata.json` patches used by the Anaconda.org to amend the metadata coming from the published packages.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-repodata-patches) via [`conda-forge/conda-forge-repodata-patches-feedstock`](https://github.com/conda-forge/conda-forge-repodata-patches-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+#### conda-forge-ci-setup
+
+This special feedstock provides a package that defines the logic to install and configure a common CI setup across providers.
+
+- ⚙️ Deployed in [Anaconda.org](https://anaconda.org/conda-forge/conda-forge-ci-setup) via [`conda-forge/conda-forge-ci-setup-feedstock`](https://github.com/conda-forge/conda-forge-ci-setup-feedstock)
+- 🔒 Has access to Azure, Anaconda.org (cf-staging)
+
+#### regro/cf-graph-countyfair
 
-<a id="smithy"></a>
+This is the graph data used by [`autotick-bot`](#autotick-bot).
 
-### Smithy
+- ⚙️ Deployed in [Github Actions via `regro/cf-graph-countyfair`](https://github.com/regro/cf-graph-countyfair)
+- ⛓ Needs [`regro/cf-scripts`](#regrocf-scripts), [`conda-forge/conda-forge-pinning-feedstock`](#conda-forge-pinning)
+- 🤖 Uses [`@regro-cf-autotick-bot`](#bot-accounts)
+- 🔒 Has access to Github API
 
-Smithy contains maintenance code for conda-forge, which is used by the `conda-smithy` command line tool and the [Admin web services](#dev-admservice). Smithy lives in the repository [conda-forge/conda-smithy](https://github.com/conda-forge/conda-smithy).
+The logic to build the graph is provided by [`cf-scripts`](#regrocf-scripts).
+
+#### regro/conda-suggest-conda-forge
+
+- [`regro/conda-suggest-conda-forge`](https://github.com/regro/conda-suggest-conda-forge) provides [`conda-suggest`](https://github.com/conda-incubator/conda-suggest) files that map executables to package names.
+
+#### docker-images
+
+This repository builds the Docker images used to provide a unified system on all Linux builds.
+
+- ⚙️ Deployed in [`conda-forge/docker-images`](https://github.com/conda-forge/docker-images)
+- 🔒 Has access to [DockerHub](#docker-hub) and [Quay.io](#quay)
+- ⛓ Needed by `staged-recipes`, feedstocks.
+
+### Code repositories
+
+These are repositories that hold programs and other codes that define behavior.
+However, their actions are often not triggered here, but rather used by other parts of the conda-forge ecosystem.
+
+#### Smithy
+
+This is the main feedstock creation and maintenance tool.
+
+- 📜 Source at [`conda-forge/conda-smithy`](https://github.com/conda-forge/conda-smithy)
+- 📦 Packaged at [`conda-forge/conda-smithy-feedstock`](https://github.com/conda-forge/conda-smithy-feedstock)
+- 📖 [Documentation](https://github.com/conda-forge/conda-smithy/blob/main/README.md)
```

Comment:
We should mention the conda-forge docs as well since that is where all of the smithy options in the conda-forge.yml are exposed in a website.